test/wcmp/wcmp.cpp

```diff
@@ -47,27 +44,41 @@ int main( int argc, char **argv)
 }		// main
 //---------------------------------------------------------------------------
 static int filesAreDiffBinary( 	// binary compare file length and contents
```

Comment:
I don't think we are using this function anywhere in our tests (@chipbarnaby ?), so maybe this should have a TODO (MP) flag too?

Reply:
The flag is written above the function. Are there other places where you would want the tag? Like the place where the function is used.